### PR TITLE
Bulk Gatway shares mounted

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/live/data.tf [hunk 1]
```diff
@@ -54,6 +54,10 @@ data "vault_generic_secret" "bulk_gateway_ec2_data" {
   path = "applications/${var.aws_account}-${var.aws_region}/${var.category}/bulk_gateway/ec2"
 }
 
+data "vault_generic_secret" "bulk_gateway_shares_data" {
+  path = "applications/${var.aws_account}-${var.aws_region}/${var.category}/bulk_gateway/shares"
```
Comment (MartyFox) [inline groups/live/data.tf]: There is a bulk-gateway path already defined that is used for SFTP so this should use the same path.

File: groups/staging/data.tf [hunk 2]
```diff
@@ -46,6 +46,10 @@ data "vault_generic_secret" "bulk_gateway_ec2_data" {
   path = "applications/${var.aws_account}-${var.aws_region}/${var.category}/bulk_gateway/ec2"
 }
 
+data "vault_generic_secret" "bulk_gateway_shares_data" {
+  path = "applications/${var.aws_account}-${var.aws_region}/${var.category}/bulk_gateway/shares"
```
Comment (MartyFox) [inline groups/staging/data.tf]: There is a bulk-gateway path already defined that is used for SFTP so this should use the same path.

File: groups/live/data.tf (hunk 1 above)
Comment (willwatters) [reply groups/live/data.tf]: Content moved to bulk-gateway vault location

File: groups/staging/data.tf (hunk 2 above)
Comment (willwatters) [reply groups/staging/data.tf]: Content moved to bulk-gateway vault location

**Review by MartyFox — APPROVED** [review]

**Pull request merged**